openstack/containerorchestration/v1/bays/results.go

```diff
@@ -0,0 +1,42 @@
+package bays
+
+import (
+	"github.com/gophercloud/gophercloud"
+)
+
+type commonResult struct {
+	gophercloud.Result
+}
+
+// Extract is a function that accepts a result and extracts a bay resource.
+func (r commonResult) Extract() (*Bay, error) {
+	var bay *Bay
+	err := r.ExtractInto(&bay)
+	return bay, err
+}
+
+// GetResult represents the result of a get operation.
+type GetResult struct {
+	commonResult
+}
+
+// Represents a Container Orchestration Engine Bay, i.e. a cluster
+type Bay struct {
```

Comment:
I left them out because the API isn't documented and I honestly don't know what the bulk of those values are, if they are final or how they would be used. So I only included the fields that I understood and needed.

If you are okay with me adding them to the struct but leaving them undocumented, I can do that.